sgt-back-end/index.js

```diff
@@ -0,0 +1,134 @@
+// eslint-disable-next-line no-console
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+const middleWare = express.json();
+app.use(middleWare);
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+  from "grades"
+  `;
+  db.query(sql)
+    .then(result => res.status(200).json(result.rows))
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({ error: 'An unexpected error occured' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const id = req.body;
```

Comment:
This variable name would be less confusing if it were `body`. `id` usually means a number or some other unique identifier.